akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/HttpCookie.scala

```diff
@@ -87,9 +147,24 @@ final case class HttpCookie(
     if (secure) r ~~ "; Secure"
     if (httpOnly) r ~~ "; HttpOnly"
     if (extension.isDefined) r ~~ ';' ~~ ' ' ~~ extension.get
+    if (sameSite.isDefined) r ~~ "; SameSite=" ~~ sameSite.get
     r
   }
 
+  override def name(): String = this.name
+  override def value(): String = this.value
+  override def secure(): Boolean = this.secure
+  override def httpOnly(): Boolean = this.httpOnly
+
+  private[http] def expires(): Option[DateTime] = this.expires
```

Comment:
Why are these `private[http]` now? Also we should remove the extra parentheses (future Scala versions might enforce them for usages).

Reply:
Hum, yeah, they don't need to be private. I guess I was following too much on the Java API side. I guess I can declare them as `val`s and not have those methods at all.